2023/das-wg-charter.html

```diff
@@ -178,14 +178,15 @@ <h2>
         <ul>
           <li>reacting to device power status
           </li>
-          <li>preventing the screen or system from turning off
+          <li>preventing the screen from turning off under certain conditions
+          </li>
+          <li>preventing the system from turning off under certain conditions
           </li>
           <li>accessing nearby devices without physical contact
           </li>
           <li>responding to ambient light levels of the device’s environment
           </li>
-          <li>reacting to changes in motion or orientation of the hosting
-          device
+          <li>reacting to changes in motion and orientation of the hosting device
           </li>
           <li>providing location information, either of the hosting device or of a location of interest
```

Comment:
```suggestion
          <li>providing location information, either of the hosting device or a location selected by the user
```

Reply:
[Support from Reilly](https://lists.w3.org/Archives/Member/member-charters-review/2024Feb/0002.html)

Reply:
Looks good to me, please make sure the change is also reflected in the WebApps WG charter.

Reply:
> Looks good to me, please make sure the change is also reflected in the WebApps WG charter.

/cc @siusin , just in case if there is no comment at WebApps

Reply:
JIC,, @svgeesus is this change ok for your point of view?